packages/terra-application-navigation/src/common/_PopupMenu.jsx

```diff
@@ -59,6 +63,10 @@ const propTypes = {
      * Key used as react key, and returned in the onSelect
      */
     key: PropTypes.string.isRequired,
+    /**
+     * The id for the menuItem.
+     */
+    id: PropTypes.string,
```

Comment:
I am not able to find where this prop is being assigned..? do we need this prop in popMenu...???

Reply:
To follow up on this, it's used down on line 206 when iterating through the menu items.